doc/user/ngctl_guide.md

```diff
@@ -0,0 +1,89 @@
+
+# Overview
+ngctl is a terminal cmd tool for nebula-operator, it has the following commands:
```

Comment:
Is this better? `for nebula-operator` sounds like management `nebula-operator`.


`ngctl` is a terminal cmd tool for Nebula Graph managed by nebula-operator, it has the following commands: